### PR TITLE
Update arity in link to c:Endpoint.start_link/1

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: lib/phoenix/endpoint.ex [hunk 1]
```diff
@@ -218,14 +218,14 @@ defmodule Phoenix.Endpoint do
       `c:broadcast_from/4`, `c:broadcast_from!/4`, `c:local_broadcast/3`,
       and `c:local_broadcast_from/4`
 
-    * for configuration: `c:start_link/0`, `c:config/2`, and `c:config_change/2`
+    * for configuration: `c:start_link/1`, `c:config/2`, and `c:config_change/2`
```
Comment (eksperimental) [inline lib/phoenix/endpoint.ex:221]: This is the only relevant change. The rest are done by the formatter.

Comment (aaronrenner) [reply lib/phoenix/endpoint.ex:221]: Thanks @eksperimental! I don't believe we are formatting the phoenix codebase yet. Would you mind removing the rest of the formatting changes?

Comment (eksperimental) [reply lib/phoenix/endpoint.ex:221]: done

**Review by aaronrenner — APPROVED** [review]
Thanks @eksperimental!

**Pull request merged**